ui/src/lib/api.ts

```diff
@@ -60,7 +60,7 @@ export const patch = async (url: string, body: Interfaces.JSON, token?: string |
     return response;
 };
 
-export const put = async (url: string, body: Interfaces.JSON, token: string | undefined): Promise<AxiosResponse> => {
+export const put = async (url: string, body: any, token: string | undefined): Promise<AxiosResponse> => {
```

Comment:
can we not keep this more specific?

Reply:
to be honest the api.js uses a pattern I'm not massively keen on (why does UI specific code have URLs for the API in it, why not have specific methods here? e.g putReferences, putPublication, seems lazy) so perhaps we dont address here

Reply:
We were having trouble getting the `Interfaces.JSON` to accept an array instead of an object so we used `any`, I've just checked to see if we can do it a different way but it's still rejecting it. But no problem we can always revisit in the future 🙂

Reply:
this will be changed. we just forgot it when we tested the update. There is more stuff into the PR that will be removed